content/en/docs/reference/glossary/affinity.md

```diff
@@ -0,0 +1,19 @@
+---
+title: affinity
+id: affinity
+date: 2019-01-11
+full_link: /docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity/
+short_description: >
+  Node affinity is a set of rules used by the scheduler to determine where a pod can be placed.
+aka:
+tags:
+- core object
```

Comment:
I'm afraid that `core object` is incorrect; there is no `Affinity` object in the Kubernetes API. You should omit this line @mehabhalodiya

Reply:
Today I learned!